src/components/CardComponent.css

```
@@ -0,0 +1,56 @@
+.cardComponent {
```

보통 css 네이밍의 컨벤션은 `소문자`, `_` , `-` 의 조합으로 하는 경우가 많습니다. (관습적인 부분이라 특별한 이유는 없어 강제 사항은 아닙니다 ㅎㅎ)
```suggestion
.card-component {
```